java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -95,4 +95,19 @@ public interface SetBaseCommands {
      * }</pre>
      */
     CompletableFuture<Boolean> sismember(String key, String member);
+
+    /**
+     * Stores the members of the set resulting from the union of all the given sets into new set.
```

Comment:
Stores the members of the union of all given sets specified by `keys` into a new set at `destination`.